egs2/TEMPLATE/mt1/mt.sh

```diff
@@ -1215,7 +1215,7 @@ if ! "${skip_eval}"; then
             detokenizer.perl -l ${tgt_lang} -q < "${_scoredir}/hyp.trn" > "${_scoredir}/hyp.trn.detok"
 
             if [ ${tgt_case} = "tc" ]; then
-                echo "Case sensitive BLEU result (single-reference)" >> ${_scoredir}/result.tc.txt
+                echo "Case sensitive BLEU result (single-reference)" > ${_scoredir}/result.tc.txt
```

Comment:
Change `>>` to `>` so that the previous result will be over-written if we re-run the inference using the same tag.